plottr/node/dim_reducer.py

```diff
@@ -171,8 +171,9 @@ def addDimension(self, name: str) -> None:
                 for o in opts:
                     combo.addItem(o)
 
+        scaling = np.rint(self.logicalDpiX() / 96.0)
         combo.setMinimumSize(50, 22)
```

Comment:
what about these?
```suggestion
        combo.setMinimumSize(50, 22 * scaling)
```

Reply:
Yes we should probably scale the MinimumSize too (both width and height 50 and 22 respectively)

Reply:
It is not necessary actually. The sizes of dropbox and slider are scaling automatically because of plot window size scaling I think. It is setMaximumHeight that was restricting this scaling. Hence an update there.

Reply:
I agree that it's probably not necessary but it is more correct. Setting the max size basically prevents this from shrinking to less that one row if it is somehow empty